openpype/hosts/fusion/plugins/publish/collect_instances.py

```diff
@@ -57,6 +57,18 @@ def process(self, instance):
             start_with_handle = comp_start
             end_with_handle = comp_end
 
+        frame = instance.data["creator_attributes"].get("frame")
+        # explicitly publishing only single frame
+        if frame:
```

Comment:
Does this mean frame 0 can never be exported? Should this be `if frame is not None`?

Reply:
Moved comment to https://github.com/ynput/OpenPype/pull/6057